lib/leaflet/runalyze.leaflet.layers.js

```diff
@@ -52,47 +63,75 @@ RunalyzeLeaflet.getNewLayers = function(){
 			}
 		),
 		'HERE': L.tileLayer(
-			'http://{s}.maptile.lbs.ovi.com/maptiler/v2/maptile/newest/hybrid.day/{z}/{x}/{y}/256/png8?token={token}&app_id={appId}', {
+			'http://{s}.maptile.lbs.ovi.com/maptiler/v2/maptile/newest/hybrid.day/{z}/{x}/{y}/' + (L.Browser.retina? '512': '256') + '/png8?token={token}&app_id={appId}',
+			{
 				subdomains: '1234',
 				appId: Runalyze.Options.nokiaAuth().app,
 				token: Runalyze.Options.nokiaAuth().token,
 				attribution: 'Map &copy; <a href="http://developer.here.com">Nokia</a>, Data &copy; NAVTEQ 2012'
 			}
 		),
+		'MapboxOutdoor': L.tileLayer(
+			'//api.mapbox.com/styles/v1/mapbox/outdoors-v10/tiles/256/{z}/{x}/{y}' + (L.Browser.retina? '@2x': '') + '?access_token={accesstoken}',
+			{
+				accesstoken: Runalyze.Options.MapboxAuth().token,
```

Comment:
This should be `Runalyze.Options.MapboxAuth()` without `.token` as there's only a single api key and no object with `app` and `token` as for `nokiaAuth()`.